extensions/wikia/Listusers/SpecialListusers_helper.php

```diff
@@ -239,19 +244,19 @@ public function loadData() {
 					}
 
 					if ( $oUTitle instanceof Title ) {
-						$links[0] = $sk->makeLinkObj( $oUTitle, $wgLang->ucfirst(wfMsg($msg) ) );
+						$links[0] = $sk->makeLinkObj( $oUTitle, $lang->ucfirst(wfMsg($msg) ) );
 					}
 
-					if ( $wgUser->isAllowed( 'block' ) && ( !$userIsBlocked ) ) {
+					if ( $user->isAllowed( 'block' ) && ( !$userIsBlocked ) ) {
 						$links[] = $sk->makeLinkObj(
 							Title::newFromText( "BlockIP/{$oUser->getName()}", NS_SPECIAL ),
-							$wgLang->ucfirst( wfMsg('blocklink') )
+							$lang->ucfirst( wfMsg('blocklink') )
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this legacy wfMsg* function call with one of the new i18n functions. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3ALegacyWfMsgUsage)

[project-id]: wikia:app